rnd/autogpt_server/autogpt_server/executor/scheduler.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+from autogpt_server.util.service import AppService, expose
+
+
+class ExecutionScheduler(AppService):
```

Comment:
This is the Example-2 of introducing a "microservice" you just need to create a class that extends (AppService), then expose the function using @expose decorator.